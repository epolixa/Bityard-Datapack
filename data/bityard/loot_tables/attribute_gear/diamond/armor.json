{
	"pools": [
		{
			"rolls": 1,
			"entries": [
				{
					"type": "loot_table",
					"name": "bityard:attribute_gear/diamond/armor/healthy"
				},
				{
					"type": "loot_table",
					"name": "bityard:attribute_gear/diamond/armor/lucky"
				},
				{
					"type": "loot_table",
					"name": "bityard:attribute_gear/diamond/armor/strong"
				},
				{
					"type": "loot_table",
					"name": "bityard:attribute_gear/diamond/armor/sturdy"
				},
				{
					"type": "loot_table",
					"name": "bityard:attribute_gear/diamond/armor/swift"
				}
			]
		}
	]
}