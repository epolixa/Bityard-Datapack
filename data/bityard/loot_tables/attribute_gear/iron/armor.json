{
	"pools": [
		{
			"rolls": 1,
			"entries": [
				{
					"type": "loot_table",
					"name": "bityard:attribute_gear/iron/armor/healthy"
				},
				{
					"type": "loot_table",
					"name": "bityard:attribute_gear/iron/armor/lucky"
				},
				{
					"type": "loot_table",
					"name": "bityard:attribute_gear/iron/armor/strong"
				},
				{
					"type": "loot_table",
					"name": "bityard:attribute_gear/iron/armor/sturdy"
				},
				{
					"type": "loot_table",
					"name": "bityard:attribute_gear/iron/armor/swift"
				},
				{
					"type": "loot_table",
					"name": "bityard:attribute_gear/iron/armor_chain/healthy"
				},
				{
					"type": "loot_table",
					"name": "bityard:attribute_gear/iron/armor_chain/lucky"
				},
				{
					"type": "loot_table",
					"name": "bityard:attribute_gear/iron/armor_chain/strong"
				},
				{
					"type": "loot_table",
					"name": "bityard:attribute_gear/iron/armor_chain/sturdy"
				},
				{
					"type": "loot_table",
					"name": "bityard:attribute_gear/iron/armor_chain/swift"
				}
			]
		}
	]
}