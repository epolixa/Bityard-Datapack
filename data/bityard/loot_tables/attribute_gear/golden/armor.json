{
	"pools": [
		{
			"rolls": 1,
			"entries": [
				{
					"type": "loot_table",
					"name": "bityard:attribute_gear/golden/armor/healthy"
				},
				{
					"type": "loot_table",
					"name": "bityard:attribute_gear/golden/armor/lucky"
				},
				{
					"type": "loot_table",
					"name": "bityard:attribute_gear/golden/armor/strong"
				},
				{
					"type": "loot_table",
					"name": "bityard:attribute_gear/golden/armor/sturdy"
				},
				{
					"type": "loot_table",
					"name": "bityard:attribute_gear/golden/armor/swift"
				}
			]
		}
	]
}